src/reaper_osara.cpp

```diff
@@ -849,6 +849,29 @@ string gridDivisionToFriendlyName(double division) {
 	return translate("grid  unknown");
 }
 
+TimeFormat getPrimaryOrSecondaryTimeFormatForCommand() {
+	TimeFormat tf;
```

Comment:
A variable is redundant here because we can just return from the function. Specifically:

1. Remove this variable.
2. Make all the tf = lines into return lines.
3. Remove the else statements, since else is unnecessary after return.
4. Return TF_RULER as a fallback at the bottom, but we should never hit that.